custom-service-adapter/knotx-standalone-1.0.0.json

```diff
@@ -0,0 +1,26 @@
+{
+  "modules": [
+    "knotx:io.knotx.KnotxServer",
+    "knotx:io.knotx.FilesystemRepositoryConnector",
+    "knotx:io.knotx.FragmentSplitter",
+    "knotx:io.knotx.FragmentAssembler",
+    "knotx:io.knotx.ServiceKnot",
+    "knotx:io.knotx.HandlebarsKnot",
+    "knotx:io.knotx.example.CustomServiceAdapter"
+  ],
+  "config": {
+    "knotx:io.knotx.ServiceKnot": {
+      "options": {
+        "config": {
+          "services": [
+            {
+              "name": "booksdata",
+              "address": "knotx.adapter.service.custom",
+              "params": {}
```

Comment:
No, there is a NPE in Knot.x when no params are defined.
See: https://github.com/Cognifide/knotx/issues/282